hermes/crates/cbork/cddl-parser/tests/byte_sequences.rs

```diff
@@ -5,12 +5,61 @@ use cddl_parser::{
     cddl_test::{CDDLTestParser, Parser, Rule},
 };
 
+pub const HEXPAIR_PASSES: &[&str] = &[
```

Comment:
Are these consts used in any other places besides the corresponding unit tests ?

If no, I think this data should be defined inside the unit tests, without any public scope

Reply:
They can be used between the tests, and these are integration tests.

Reply:
They should be `pub(crate)` however.
The compiler should eliminate and data which isn't actually referenced during the link, so it shouldn't matter.